> Disables `import/no-extraneous-dependencies` in `client/webpack.config.*`. Ideally, we need a mechanism to declare that a file is a build script that can import Node.js modules and the globally declared `devDependencies`.

I just pushed df7d57ee7458074b3a9ae6486a5db1641c59731c that uses inline eslint config so the webpack config looks at the root. It's one way of dealing with this issue.

It exposes this issue:

```
…/client/webpack.config.js
  18:39  error  'duplicate-package-checker-webpack-plugin' should be listed in the project's dependencies. Run 'npm i -S duplicate-package-checker-webpack-plugin' to add it  import/no-extraneous-dependencies

✖ 1 problem (1 error, 0 warnings)
``

--- a/client/webpack.config.js
+++ b/client/webpack.config.js
@@ -1,5 +1,5 @@
 /**
- **** WARNING: No ES6 modules here. Not transpiled! ****
+ * **** WARNING: No ES6 modules here. Not transpiled! ****
  */
 
 /* eslint import/no-extraneous-dependencies: [ "error", { packageDir: __dirname/.. } ] */

--- a/client/webpack.config.node.js
+++ b/client/webpack.config.node.js
@@ -2,8 +2,8 @@
  * **** WARNING: No ES6 modules here. Not transpiled! ****
  */
 
+/* eslint import/no-extraneous-dependencies: [ "error", { packageDir: __dirname/.. } ] */
 /* eslint-disable import/no-nodejs-modules */
-/* eslint-disable import/no-extraneous-dependencies */
 
 /**
  * External dependencies

--- a/package.json
+++ b/package.json
@@ -180,6 +180,7 @@
 		"circular-dependency-plugin": "5.2.0",
 		"concurrently": "5.0.1",
 		"cross-env": "6.0.3",
+		"duplicate-package-checker-webpack-plugin": "3.0.0",
 		"enzyme": "3.11.0",
 		"enzyme-adapter-react-16": "1.15.1",
 		"enzyme-to-json": "3.4.3",

